app/main/routes_frontend.py

```diff
@@ -106,19 +107,19 @@ def import_zymatic_recipe():
         machine = next((uid for uid in active_brew_sessions if not active_brew_sessions[uid].is_pico), None)
         try:
             sync_user_uri = 'http://137.117.17.70/API/SyncUSer?user={}&machine={}'.format(guid, machine)
-            current_app.logger.debug('DEBUG: import_zymatic_recipe - {}'.format(sync_user_uri))
+            current_app.logger.debug('import_zymatic_recipe - {}'.format(sync_user_uri))
```

Comment:
Use strings from recipe_import?

Reply:
yeah I want to refactor all the imports to use `recipe_import` instead 👍

Reply:
should probably have 137.117.17.70 as a single variable but 🤷‍♂️

Reply:
refactored everything to just use recipe_import.py instead